package.json

```diff
@@ -7,7 +7,8 @@
     "lint": "eslint \"src/**/*.js\" \"test/**/*.js\"",
     "test": "npm run lint && npm run build",
     "build": "webpack -p",
-    "start": "webpack-dev-server -d --inline --hot"
+    "start": "webpack-dev-server -d --inline --hot",
+    "prepublish": "babel -d dist/ src/"
```

Comment:
This will transform ES6 sources to ES5 with CommonJS imports, so it's a "legal" NPM module (see #122); it should target `lib/` then, not `dist/` (which contains the output from `webpack`).

`main` above should then be updated to point to `lib/Main.js`.

And `.npmignore` should ignore `dist/` (we'll publish the ES5 sources to NPM, not the output from `webpack`), and `.gitignore` should also ignore `lib/`.

(see #44 where this was all done)